NEWS

```diff
@@ -12,8 +12,155 @@
+ * Fixed missing reg_id on memh creation
+ * Fixed repeated invalidations by retaining existing access flags
+ * Fixed abort reason propagation for rendezvous RTR mtype
+ * Fixed transport availability for configuration list in ALLOW mode
+ * Fixed wrong flag being used for checking BCOPY capability
+ * Fixed sending too many ATPs for small messages
+ * Enforced 16 bits size for active messages identifiers
+ * Fixed unnecessary status check for emulated AMO
+ * Fixed more than one fragment sending in rendezvous pipeline
+ * Fixed crash by using biggest max frag across all lanes
+ * Fixed missing memory handle flags by copying from parent to child
+ * Fixed worker interface activate count
+ * Fixed flush requests by replacing ATP/flush lane map with lane indexes
+ * Fixed lost uct_flags when merging memory regions
+#### UCT
+ * Fixed memory domain UCT flags description
+#### RDMA CORE (IB, ROCE, etc.)
+ * Fixed FETCH_ADD remote access error for ODP/KSM case
+ * Fixed missing conditional compilation checks for DM
+ * Fixed IB MD allocation naming typo
+ * Fixed invalid GIDs filter in IB
+ * Fixed flags usage in MLX5 zcopy_post
+ * Do not limit ODP registration retries
+ * Fixed JUCX failures by considering the number of supported completion vectors
+#### CUDA
+ * Fixed async memory handling using cuda memory type on Grace
+ * Added rcache overhead in performance estimation
+ * Fixed GDR copy performance regression by providing maximum estimation between get and put
+ * Fixed CUDA IPC reachability check
+ * Fixed crash in MPI_Finalize when CUDA context is destroyed
+ * Always require rcache by default for GDR
+ * Fixed md_close when no rcache
+ * Fixed CUDA copy memory domain allocations
+ * Fixed multiple tests for gdr_copy transport
+#### UCS
+ * Fixed a crash by using heap allocation to process expired timers in batch
+ * Fixed allocation issue on memtrack dump
+ * Fixed deletion of the monitored folder in VFS
+ * Fixed unsafe resize for DC initiator array
+ * Fixed function macro invocation to match C standard
+ * Fixed calling async handler on already released resource
+ * Fixed performance by setting higher bandwidth for different NUMA nodes on Grace
+ * Fixed undeclared value error in timer conversion routine
+ * Fixed uninitialized value access in registration cache
+#### UCM
+ * Fixed race condition in parsing proc maps
+ * Fixed mremap failure while parsing /proc/self/maps
+#### ROCM
+ * Fixed ROCM interface reachability test
+ * Fixed memory domain fork test
+#### TCP
+ * Always bind endpoint to interface
+#### Tools
+ * Fixed buffer size potential overflow in ucx_perftest
+ * Fixed missing address when packing memory keys on ucx_perftest
+ * Fixed memory leak for endpoint report in ucx_info
+ * Fixed build without openmp in ucx_perftest
+ * Fixed UCT device override on server side on ucx_perftest
+#### Build
+ * Fixed using correct ASAN version for running tests
+#### Configuration
+ * Fixed Intel compiler options support checks
```

Comment:
The line can be removed since we have: `Feature/Build: Improved Intel Compiler detection and support`

Reply:
fixed